artemis-console-extension/artemis-extension/src/artemis-extension/artemis/artemis-service.ts

```diff
@@ -589,6 +595,20 @@ class ArtemisService {
     checkCanBrowseQueue = (queueMBean: MBeanNode ): boolean => {
         return (this.DEBUG_PRIVS && queueMBean?.hasInvokeRights(BROWSE_SIG)) ?? false;
     }
+
+    doesCopyMessageMethodExist = (broker: MBeanNode | undefined, queue: string): boolean => {
+        return this.doesMethodExist(broker, queue, COPY_MESSAGE_SIG);
+    }
+
+    doesMethodExist = (broker: MBeanNode | undefined, queue: string, method: string): boolean => {
+        if(broker) {
+            var queueMBean = broker.parent?.find(node => { 
+                return node.propertyList?.get('subcomponent') === 'queues' && node.name === queue 
+            })
+            return queueMBean?queueMBean.hasOperations(method): false;
+        }
+        return false;
+    }
```

Comment:
Could there be a situation where there are several `queueMBean` validating the condition in the find? If so, I would change the find to a filter and try to find the operation in at least one of them.
If not, ignore this

Reply:
nope that cant happen as its a 1 to 1 with address name and queue name